### PR TITLE
Add review_text field to reviews

### LINKED ISSUE #194: Reviews: add review text

### PR DESCRIPTION
**Pull request by Rhantolq**
Max review_text length is 255 @jakubpw
All currently avaible reviews have "" as their review text.
Closes #194

### CODE REVIEW

**Comment by kantoniak** [comment]
@Rhantolq please use at least one of the following languages in your review: Chinese, Japanese, Korean, Vietnamese, Thai, Cambodian, Arabic (any dialect), Persian, Hebrew, Mongolian, Hindi, Amharic.
I will update the list if I come up with something new :smiley:

**Comment by avan1235** [comment]
> Could you add this field to API too?

@kantoniak I think it was added via FixedMenuOptionReviewSerializer field

**Comment by kantoniak** [comment]
Yes, it's in there already. My bad.

File: django-server/server/admin/management/commands/generate_reviews.py [hunk 1]
```diff
@@ -2,10 +2,98 @@
...
+    "Could've been worse",
+    "I enjoyed it",
+    "Salty",
+    "Loved the food, will come again.",
+    "Splendid, absolutely refined taste.",
+    "Kinda tasteless...",
+    "The taste was pretty lit if you'd ask me. The main dish's flavour was rockin'. Gonna definitely drop here more!",
+
+    "Delicious 😋",
+
```
Comment (kantoniak) [inline django-server/server/admin/management/commands/generate_reviews.py:21]: Extra newlines? In following lines too.

File: django-server/server/admin/management/commands/generate_reviews.py [hunk 2]
```diff
@@ -2,10 +2,98 @@
...
+    "Smakowało :)",
+    "Niedobre",
+    "Nie było złe, ale nie urywa",
+    "Po pierwsze, co ja w ogóle zjadłem to ja nawet nie wiem. Koszmar! Ziemniaki niedogotowane,"  # continues
+    " mięso twarde! Jestem zawiedziony i chcę zwrotu.",
+
+    ":<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<"
+    "<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<"
+    "<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<"
+    "<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<",
```
Comment (kantoniak) [inline django-server/server/admin/management/commands/generate_reviews.py:42]: `# continues` for cohesion with earlier

File: django-server/server/admin/management/commands/generate_reviews.py [hunk 3]
```diff
@@ -2,10 +2,98 @@
...
+    "Gabrysia",
+    "a.pestka",
+    "dark_michał",
+    "KaLorakkk",
+    "p4v31",
+    "1gor",
+    "david_504",
+    "xXx_Kucharz_xXx",
+    "Oleksandra",
+    "Robert'); DROP TABLE Camera_Event;--",
```
Comment (kantoniak) [inline django-server/server/admin/management/commands/generate_reviews.py:68]: :smile:

Comment (avan1235) [reply django-server/server/admin/management/commands/generate_reviews.py:68]: that's not the name of table in our model. let's try with api_cameraevent xd

**Pull request merged**